config/env/development.js

```diff
@@ -60,5 +60,6 @@ module.exports = {
 				pass: process.env.MAILER_PASSWORD || 'MAILER_PASSWORD'
 			}
 		}
-	}
+	},
+  livereload: true
```

Comment:
That page is massively messed up in terms of spacing. Even certain lines are mixed between spacing and tabs so Sublime is confused. Should be fixed now.